### PR TITLE
Active Scan stats

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: zap/src/main/java/org/parosproxy/paros/core/scanner/PluginStats.java [hunk 1]
```diff
@@ -38,11 +42,11 @@
     /**
      * Constructs a {@code PluginStats}.
      *
-     * @param pluginName the name of the plugin.
      * @see #start()
      */
-    PluginStats(String pluginName) {
-        this.pluginName = pluginName == null ? "" : pluginName;
+    public PluginStats(Plugin plugin) {
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/core/scanner/PluginStats.java:47]: Keep package? External code should not need to create instances.

Comment (psiinon) [reply zap/src/main/java/org/parosproxy/paros/core/scanner/PluginStats.java:47]: Thats weird, I didnt mean to change that :/

Comment (psiinon) [reply zap/src/main/java/org/parosproxy/paros/core/scanner/PluginStats.java]: Fixed.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**